### PR TITLE
veusz: fix update

### PR DESCRIPTION
**Pull request by Kiri2002**
## Involved packages

 - veusz

## Involved issue

fixed update from regex to use_max_tag

## Details

- [x] Tested in the local machine (largest 16G RAM) and it is passed without any issue
- [ ] Provide New Package
- [x] Fix the Packages
  - [x] PKGBUILD
  - [x] lilac.yaml
  - [ ] lilac.py
- [ ] Would like to continue to work with us

### CODE REVIEW

**Comment by Kiri2002** [comment]
[veusz-3.6.2-2-x86_64.pkg.tar.zst-namcap.log](https://github.com/BioArchLinux/Packages/files/12377387/veusz-3.6.2-2-x86_64.pkg.tar.zst-namcap.log)
[PKGBUILD-namcap.log](https://github.com/BioArchLinux/Packages/files/12377388/PKGBUILD-namcap.log)
[veusz-3.6.2-2-x86_64-build.log](https://github.com/BioArchLinux/Packages/files/12377389/veusz-3.6.2-2-x86_64-build.log)

**Comment by pekkarr** [comment]
A RFC about [not using Pypi sources if there are alternatives](https://gitlab.archlinux.org/archlinux/rfcs/-/blob/master/rfcs/0020-sources-for-python-packaging.rst) was approved and merged into the Arch Linux RFC repo last week. Thus, I think it would be better to not convert packages to using Pypi. I don't have a strong opinion on this though.

**Comment by Kiri2002** [comment]
> 

I see, continue to use Github source, fixed update from regex to use-max-tag

**Comment by Kiri2002** [comment]
> A RFC about [not using Pypi sources if there are alternatives](https://gitlab.archlinux.org/archlinux/rfcs/-/blob/master/rfcs/0020-sources-for-python-packaging.rst) was approved and merged into the Arch Linux RFC repo last week. Thus, I think it would be better to not convert packages to using Pypi. I don't have a strong opinion on this though.

I see, continue github source, fixed from regex to use-max-tag

File: BioArchLinux/veusz/lilac.yaml [hunk 1]
```diff
@@ -1,14 +1,14 @@
-build_prefix: extra-x86_64
 maintainers:
-- github: kiri2002
-  email: kiri@vern.cc
+  - github: kiri2002
+    email: kiri@vern.cc
+build_prefix: extra-x86_64
 pre_build_script: |
-  update_pkgver_and_pkgrel(_G.newver.lstrip('v'))
+  update_pkgver_and_pkgrel(_G.newver.lstrip('veusz-'))
```
[the remark this answers is not in the document]
Comment (hubutui) [reply BioArchLinux/veusz/lilac.yaml:6]: @lilydjwg You mean if we specify `prefix`, then there is no need to use this anymore?

[the remark this answers is not in the document]
Comment (hubutui) [reply BioArchLinux/veusz/lilac.yaml:6]: Nice! see also https://nvchecker.readthedocs.io/en/latest/usage.html#global-options

[the remark this answers is not in the document]
Comment (Kiri2002) [reply BioArchLinux/veusz/lilac.yaml:6]: so should be
```
update_pkgver_and_pkgrel(_G.newver)
```
right ?

**Pull request merged**